### DISCUSSION TITLE
More

### DISCUSSION THREAD

**Discussion by Sqrts321** [community/community#140436] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello :3
I go by squirtle or seek (don't question it)
I didn't have enough room in my bio so I made this discussion... I'm new and don't really know how this works :(

also: What is the email verification rate limit? Kind of need to confirm my actual email

**Comment by Sqrts321** [comment]
looking forward to learning more about GitHub

**Comment by KavinMadusanka** [comment]
If you are referring to the rate limit for how many times you can request email verification, it's not explicitly mentioned by GitHub, but generally, if you're locked out after too many attempts, it's best to wait an hour or two before trying again
